### PR TITLE
Send SMS directly (#57)

### PR DESCRIPTION
**Pull request by SilkyPantsDan**
Addresses issue #57 to allow the user to send an SMS message directly

### CODE REVIEW

**Review by zbarbuto — APPROVED** [review]

**Comment by muhammed3ly** [comment]
please merge this 😆

File: lib/src/flutter_sms_platform.dart [hunk 1]
```diff
@@ -32,9 +32,12 @@ class FlutterSmsPlatform extends PlatformInterface {
     _instance = instance;
   }
 
+  ///
+  ///
   Future<String> sendSMS({
     required String message,
     required List<String> recipients,
+    required bool sendDirect,
```
Comment (zbarbuto) [inline lib/src/flutter_sms_platform.dart:40]: This should probably be optional and default to false so it's not a breaking change.

**Comment by zbarbuto** [comment]
@rodydavis sorry not sure who to ping on this but you were the last one to merge a PR in this repo if you could PTAL 🙏

**Comment by sigma-libra** [comment]
@rodydavis Commenting to say this would be amazing to have, and you seem like the one to talk to to get things merged around here. Any chance we can get this feature soon?

**Comment by rodydavis** [comment]
Reviewing now

**Comment by rodydavis** [comment]
Does sending directly on android require the permission in the manifest? 

In other words can it be opted out of?

I have had apps rejected because I used the API and wasn't considered a phone messaging app

**Comment by zbarbuto** [comment]
@rodydavis yes it does require that permission - we've had apps rejected ourselves for the same reason - the category for apps that are allowed to do this is very narrow.

I've made the default for this argument `false` and added a warning to the readme about using the feature.

The feature is more "opt-in" than needing to "opt-out" as if you do not add that manifest property and never call with `sendDirect: true` it is never used.

**Comment by rodydavis** [comment]
Ok that is wonderful to hear! I will merge this and publish soon!

I just wanted to make sure we don't have a lot of apps rejected over a version bump.

**Comment by rodydavis** [comment]
Published! https://pub.dev/packages/flutter_sms

**Pull request merged**